Dnn.AdminExperience/EditBar/Dnn.EditBar.Library/Dnn.EditBar.Library.csproj

```diff
@@ -86,6 +83,7 @@
     </AdditionalFiles>
   </ItemGroup>
   <ItemGroup>
+    <None Include="app.config" />

```

Comment:
We don't have this `app.config`, right?
```suggestion
```

Reply:
Correct, just removed it from the project